plugins/GSdx/GSRenderer.cpp

```diff
@@ -221,22 +221,27 @@ bool GSRenderer::Merge(int field)
 		src[i] = GSVector4(r) * scale / GSVector4(tex[i]->GetSize()).xyxy();
 		src_hw[i] = (GSVector4(r) + GSVector4 (0, y_offset[i], 0, y_offset[i])) * scale / GSVector4(tex[i]->GetSize()).xyxy();
 
-		GSVector2 off(0, 0);
+		GSVector2 offset(0, 0);
+		GSVector2i diff(dr[i].left - baseline.x, dr[i].top - baseline.y);
 
 		// Time Crisis 2/3 uses two side by side images when in split screen mode.
-		off.x = tex[i]->GetScale().x * (dr[i].left - baseline.x);
+		// Though ignore cases where baseline and display rectangle offsets only differ by 1 pixel, causes blurring and wrong resolution output on FFXII
+	    if(diff.x >= 2)
+		{
+			offset.x = tex[i]->GetScale().x * diff.x;
+		}
 
-		if(dr[i].top - baseline.y >= 4) // 2?
+		if(diff.y >= 4) // Shouldn't this be 2?
 		{
-			off.y = tex[i]->GetScale().y * (dr[i].top - baseline.y);
+			offset.y = tex[i]->GetScale().y * diff.y;
 
-			if(m_regs->SMODE2.INT && m_regs->SMODE2.FFMD)
+			if(isinterlaced() && m_regs->SMODE2.FFMD)
```

Comment:
I think you named the function incorrectly when you introduced it - `IsInterlaced` fits the naming convention better.